www/js/app.js

```diff
@@ -117,7 +120,10 @@ define(['jquery', 'zimArchiveLoader', 'uiUtil', 'settingsStore','abstractFilesys
             }, 100);
         }
     }
-    $(document).ready(resizeIFrame);
+    $(document).ready(function() {
+        resizeIFrame();
+        switchHomeKeyToFocusSearchBar();
```

Comment:
@Hrishi-Raj I don't think you need to run this in the `document.ready` function. Just launch `switchHomeKeyToFocusSearchBar()` after line 86 above.